backend/src/routes/users.js

```diff
@@ -65,7 +65,10 @@ const createUser = async (req, res) => {
         } else {
             const token = randtoken.generate(16);
             user.token = token;
+            const pin = randtoken.generate(4, '0123456789');
+            user.pin = pin;
             const createdUser = await db.users.createUser(user);
+            console.log(`User Pin Generated: ${pin}`);
```

Comment:
There's now too logs about the PIN generation. Here and below in `users.js`. I am not sure if we should be logging the new PIN as it is the authentication method now. Instead, it would be good to only log the resetting like `PIN was reseted for user with phone number +3581234567`.